.goreleaser.yaml

```diff
@@ -0,0 +1,66 @@
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
+env:
+  - VersioningPath={{ "github.com/celestiaorg/celestia-node/nodebuilder/node" }}
+before:
+  hooks:
+    # You may remove this if you don't use go modules.
+    - go mod tidy
+builds:
+  - main: ./cmd/celestia
+    binary: celestia
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
```

Comment:
Very similar comments as for https://github.com/celestiaorg/celestia-app/pull/2437#pullrequestreview-1612900245
```suggestion
```

Reply:
Same response, cc @renaynay and @Wondertan 
Any preferences on the final OS and Arch builds? 
These were just the defaults added by `init` so happy to update as the team wants.